tests/bot/exts/info/test_information.py

```diff
@@ -614,7 +614,7 @@ async def test_return_none_if_one_rule_number_is_invalid(self):
                     str(rule_number) for rule_number in extracted_rule_numbers
                     if rule_number < 1 or rule_number > len(self.full_rules))
 
-                final_rule_numbers = await self.cog.rules(self.cog, self.ctx, *raw_user_input)
+                final_rule_numbers = await self.cog.rules(self.cog, self.ctx, raw_user_input)
```

Comment:
It's incorrect to merely not unpack it. The command expects one string now, not a tuple.

Reply:
With the new ctx inspection this seems to be the easiest way to keep the tests working. It would be more elegant to rewrite the tests to use the ctx object, but in any case, the tests aren't currently effectively testing the convert portions anyways? (Hence the issue). So I think the test rewrite that also tests the converter can come later?

Reply:
Removed the hacks for much simpler fix in [`71fdd6c` (#2310)](https://github.com/python-discord/bot/pull/2310/commits/71fdd6cfd4e5353be61a46a2ab629835238122fc)